src/views/ActivitiesAll/index.js

```diff
@@ -50,7 +49,7 @@ export default class GordonActivitiesAll extends Component {
   }
 
   async componentWillMount() {
-    this.setState({ loading: true });
+    // this.setState({ loading: true });
```

Comment:
So I decided to make my internet slow through the developer tools and see how the Involvements Tab rendered. It rendered perfectly fine with that code commented out. I'm not sure why I left it there but it looks fine because the page will display everything but show a loader for the activity list and the list of activities that you're apart of for that semester. So while the entire Involvements page may not have a loader, the places where data is fetched does. You can see this on line 301 and 307 :)

Reply:
kk